survlearners/R/csf.R

```diff
@@ -1,8 +1,6 @@
-#' @include utils.R
```

Comment:
did you accidentally include these changes in this branch? when starting a new branch remember to checkout from master, I think now you did a checkout from one of your other branches?

Reply:
yes, I think I did this earlier. Now I am paying more attention to switching branches for different tasks. Sorry about the confusion, getting better with it now

Reply:
good job! just takes a bit of practice to form muscle memory

Reply:
Exactly, thanks!

Reply:
Ok, just to clarify, when I switch to a new task, I should do: git checkout master -> git branch newbranch -> git checkout newbranch -> git commit . -> add notes -> git push origin newbrach, correct?

Reply:
yes!

[git branch newbranch + git checkout newbranch can also be simplified to git checkout -b newbranch]

Reply:
and remember to git pull master first so you have the latest

Reply:
great, thanks a lot!

Reply:
git pull origin master instead of git pull master?

Reply:
just git pull master should work for you but you can be specific and say origin if it gives you a warning

Reply:
@erikcs It says "Review required" and "Merging is blocked" below....

Reply:
What's the question?

Reply:
I thought these two messages mean that someone needs to review my code and approve it for me, am I wrong?

Reply:
My question is why you have these 3 PRs open when all the changes are being made in https://github.com/som-shahlab/survmetabench/pull/44

do you want to restart it from scratch and practice your git skills and make 3 disjoint changes in 3 branches or just merge https://github.com/som-shahlab/survmetabench/pull/44 ?

Reply:
I see your point. I can just merge #44 and close the other ones. I don't want to remake those changes at this point. Can we just merge #44?

Reply:
Yes, approved. Remember to delete the old branches from github if you dont need them anymore!

Reply:
Ok, will do, thanks!